README.md

```diff
@@ -65,69 +35,4 @@ Autonomous services that use Autonolas technology are implemented as [Finite Sta
 
 ## Running the service
```

Comment:
```suggestion
## Running a demo the service
```

Reply:
Place this section above, to be consistent with the sectioning order in https://docs.autonolas.network/product/iekit/

Reply:
Fixed